.github/check_agda.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+export Agda_datadir=$PWD/data
+export PATH=$PWD/bin:$PATH
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  # See https://gitlab.haskell.org/ghc/ghc/-/issues/17418
+  xattr -rc $PWD/bin
+fi
+
+echo "The nightly version of Agda is"
+agda --version
+
+echo "Checking if Agda nightly is runnable on your system..."
+for prim in ./data/lib/prim/Agda/**/*.agda
+do
+    echo "Checking "$prim
+    agda -v 2 $prim
+done
+
+if [ $? == 0 ];
+then
+  printf "$(tput bold)Agda appears to work.\nCopy the following lines to your shell profile to access the nightly build, e.g. ~/.bashrc\n"
```

Comment:
This will leave the user's terminal with **bold text**, unless they have a prompt that does the reset for them. The bolded sequence should be terminated by `$(tput sgr0)` to reset the text to normal mode.

Reply:
thanks!